File: src/main/java/br/com/dgimenes/nasapiccontentfetcher/service/api/NasaAPODWebservice.java
package br.com.dgimenes.nasapiccontentfetcher.service.api;

import br.com.dgimenes.nasapicserver.model.APOD;
import retrofit.Call;
import retrofit.http.GET;
import retrofit.http.Query;

public interface NasaAPODWebservice {

    @GET("/planetary/apod")
    Call<APOD> getAPOD(@Query("api_key") String nasaApiKey,
                       @Query("concept_tags") boolean conceptTags,
                       @Query("date") String formattedDate);
}

File: src/main/java/br/com/dgimenes/nasapiccontentfetcher/service/api/NasaAPODWebservice.kt
package br.com.dgimenes.nasapiccontentfetcher.service.api

import br.com.dgimenes.nasapicserver.model.APOD
import retrofit.Call
import retrofit.http.GET
import retrofit.http.Query

interface NasaAPODWebservice {

    @GET("/planetary/apod")
    fun getAPOD(@Query("api_key") nasaApiKey: String,
                @Query("concept_tags") conceptTags: Boolean,
                @Query("date") formattedDate: String): Call<APOD>
}
